README.md

```diff
@@ -54,26 +54,62 @@ From the root of this repository run `cargo build -p ironoxide-java`. The result
 
 # IronOxide-Android
 
+## Using IronOxide-Android
```

Comment:
@clintfred I added this section to the top-level README. Please look it over and let me know if there are things I should add/change